src/Features/Core/Diagnostics/EngineV1/DiagnosticIncrementalAnalyzer.cs

```diff
@@ -24,997 +24,998 @@ namespace Microsoft.CodeAnalysis.Diagnostics.EngineV1
 
     internal partial class DiagnosticIncrementalAnalyzer : BaseDiagnosticIncrementalAnalyzer
     {
-        private static readonly int s_stateTypeCount = Enum.GetNames(typeof(StateType)).Count();
-        private static readonly ImmutableArray<StateType> s_documentScopeStateTypes = ImmutableArray.Create<StateType>(StateType.Syntax, StateType.Document);
+            private static readonly int s_stateTypeCount = Enum.GetNames(typeof(StateType)).Count();
+            private static readonly ImmutableArray<StateType> s_documentScopeStateTypes = ImmutableArray.Create<StateType>(StateType.Syntax, StateType.Document);
```

Comment:
The formatting is making the diff hard to understand - is there any change in this file at all?